### ISSUE TITLE
UI: when displaying arguments in textarea with spaces, these should be escaped

### ISSUE THREAD

**Issue by rangoo94** [kubeshop/testkube#4728]
**Describe the bug**

When there is a test that has an argument with spaces, like:

```yaml
apiVersion: tests.testkube.io/v3
kind: Test
metadata:
  name: some-test
spec:
  type: cypress/project
  content:
  executionRequest:
    args:
      - "--testSuitePath=Test Suites/FOLDER/mytest"
```

It should correctly escape it on Test Details > Settings > Variables page.

**To Reproduce**

1. Create such test ([**example**](https://demo.testkube.dev/tests/temp-e2e-cypress-git-created-28f089f43dad45d2b740448c5c1d53ec/settings/variables))
2. Go to its "Variables & Secrets" configuration
3. Arguments textarea has `--testSuitePath=Test Suites/FOLDER/mytest` content

**Expected behavior**

The textarea should have escaped that argument from YAML spec, either with quotes or backslash:

```
"--testSuitePath=Test Suites/FOLDER/mytest"
'--testSuitePath=Test Suites/FOLDER/mytest'
--testSuitePath=Test\ Suites/FOLDER/mytest
```

**Comment by Estegar** [comment]
Hello there, will this be fixed in the near future?
I'm on Testkube:
API Versionv1.16.26
Dashboard Versionv1.16.2
Helm Chart Versionv1.16.34

This is still an issue on this version.

**Comment by vsukhin** [comment]
hey, @Estegar I will ask our frontend team to check @rangoo94 @haneabogdan

**Comment by Estegar** [comment]
I discovered a workaround for Katalon, and I believe it should work for any other testing application. However, it requires the use of a custom executor.

You can add spaces and either double ("") or single ('') quotes in a command within the executor settings, the formatting will remain the same in your test execution.

**Comment by topliceanurazvan** [comment]
Hello @Estegar,

The feature has been fixed, but not yet released. We are planning to do that as soon as possible. I will let you know when that happens.

Sorry for the inconvenience.

**Comment by Estegar** [comment]
> I discovered a workaround for Katalon, and I believe it should work for any other testing application. However, it requires the use of a custom executor.
> 
> You can add spaces and either double ("") or single ('') quotes in a command within the executor settings, the formatting will remain the same in your test execution.

This does work but only one time, next run will have broken arguments because of space, double ("") or single ('') quotes. I don't even know how it worked the first time. I can't reproduce that so it was fake news unfortunately.

**Comment by topliceanurazvan** [comment]
@Estegar we just released `v1.16.5` for the dashboard. Please make sure to update and let us know if the fix is working as expected.

**Comment by Estegar** [comment]
@topliceanurazvan Hey, unfortunately, it is not working as intended. I think the current issue might not fully adress my case.
I will create a new issue with an explanation of how to reproduce the bug. There are several issues in this area, so I will provide full explanation there. Stay tuned!
